spec/lib/hydranorth/doi_service_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'spec_helper'
+
+describe Hydranorth::DOIService do
+  let(:example_doi_id) { 'doi:10.5072/FK2JQ1003W' }
+  let(:generic_file) do
+    # TODO this is coming from sufia's factories, fix this
```

Comment:
I feel like we should have a standard generic file factory. We have before(each)/let blocks through out our test suite doing very similar logic for generating new Generic Files. Should be consolidated. Also we are including all the factories from Sufia as well. Not sure why? Here I am hooking into Sufia's generic file factory, but this should be using our own.